Sheepish-Squirrels/bangazon-api-sprint1
# Added GET, POST, PUT, and DELETE functionality to computers

Fixes #8 

Changes proposed in this pull request:

- put back in code for generating payment-type data that must have been deleted while resolving merge conflicts
- created file to generate computer data
- created faker computer data and storing into JSON file
- added functionality to add computer data to sql table
- created routes for computer data
- created promises for GET, POST, PUT, DELETE computers in model folder
- created controller for computer data to retrieve the data needed

Steps to test:

In terminal,
run nodemon app.js
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
Make sure you can GET all computers, GET one computer, POST a computer, PUT (edit) a computer, and DELETE a computer
